branding/ghostery/branding.nsi

```diff
@@ -8,21 +8,21 @@
 
 # BrandFullNameInternal is used for some registry and file system values
 # instead of BrandFullName and typically should not be modified.
-!define BrandFullNameInternal "Mozilla Developer Preview"
-!define BrandFullName         "Mozilla Developer Preview"
-!define CompanyName           "mozilla.org"
-!define URLInfoAbout          "https://www.mozilla.org"
-!define HelpLink              "https://support.mozilla.org"
+!define BrandFullNameInternal "Ghostery"
+!define BrandFullName         "Ghostery"
+!define CompanyName           "ghostery.com"
+!define URLInfoAbout          "https://www.ghostery.com"
+!define HelpLink              "https://www.ghostery.com/support/"
 
-!define URLStubDownloadX86 "https://download.mozilla.org/?os=win&lang=${AB_CD}&product=firefox-latest"
-!define URLStubDownloadAMD64 "https://download.mozilla.org/?os=win64&lang=${AB_CD}&product=firefox-latest"
-!define URLStubDownloadAArch64 "https://download.mozilla.org/?os=win64-aarch64&lang=${AB_CD}&product=firefox-latest"
-!define URLManualDownload "https://www.mozilla.org/${AB_CD}/firefox/installer-help/?channel=release&installer_lang=${AB_CD}"
-!define URLSystemRequirements "https://www.mozilla.org/firefox/system-requirements/"
-!define Channel "unofficial"
+!define URLStubDownloadX86 "https://www.ghostery.com/products/?os=win&lang=${AB_CD}&product=firefox-latest"
+!define URLStubDownloadAMD64 "https://www.ghostery.com/products/?os=win64&lang=${AB_CD}&product=firefox-latest"
+!define URLStubDownloadAArch64 "https://www.ghostery.com/products/?os=win64-aarch64&lang=${AB_CD}&product=firefox-latest"
+!define URLManualDownload "https://www.ghostery.com/products/?installer_lang=${AB_CD}"
+!define URLSystemRequirements "https://www.ghostery.com/products/"
+!define Channel "ghostery"
 
 # The installer's certificate name and issuer expected by the stub installer
-!define CertNameDownload   "Mozilla Corporation"
+!define CertNameDownload   "Ghostery Inc."
 !define CertIssuerDownload "DigiCert SHA2 Assured ID Code Signing CA"
```

Comment:
Again, this seems to be just for stub installer. Maybe we should already put in the correct values in case we want to use the stub installer in the future.